base/frontend/sourcegraph-frontend.Ingress.yaml

```diff
@@ -17,15 +17,17 @@ spec:
   # about configuring TLS
   # tls:
   # - hosts:
-  #   - example.sourcegraph.com
+  #   - sourcegraph.example.com
   #   secretName: sourcegraph-tls
   rules:
-  - http:
+  - host: sourcegraph.example.com
+    http:
       paths:
       - path: /
         backend:
           serviceName: sourcegraph-frontend
           servicePort: 30080
     # Set a host if you have multiple ingresses. This installs a default
     # handler.
+    # TODO@ggilmore: Is it safe to switch the host?
```

Comment:
what do you mean by switch the host?

Reply:
@keegancsmith I meant using an explicit host instead of relying on the default handler